### PR TITLE
Fix inconsistencies in tutorial

### PR DESCRIPTION
**Pull request by newyork-anthonyng**
## Changes I made
1. To stay consistent, I changed all color values to use `hex` values
2. Fixed minor typos
3. Added the `route/index/` directory for the tutorial. I think this might be a valuable reference for readers
4. I changed one image of the `<color-picker-header />` to display the correct color

Feel free to cherry-pick the commits and edit as you see fit.

I'm excited to see all the 💓 Marko will be getting from Reddit.

### CODE REVIEW

File: routes/index/index.marko [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<html>
```
Comment (austinkelleher) [inline routes/index/index.marko]: I didn't make it clear in the tutorial, but having an `index.marko` in the route of you project (e.g. `/marko-color-picker/index.marko`), will automatically get served as the index route's template. The tutorial does not reflect that. Perhaps you could update the tutorial to describe this? I think it's important to note that adding a `routes/index/index.marko` or `routes/some-route/index.marko` yields routes and that `marko-starter` automatically resolves routes from the `routes` folder, but ultimately, we should keep the `index.marko` at the route of the project. The Marko try-online looks for an `index.marko` at the root of each virtual project (e.g. [color-picker try-online](http://markojs.com/try-online/?file=%2Fmarko-color-picker%2Findex.marko&gist=)), so we should leave it there. Can you update the tutorial to describe this and remove this file? Thanks!

**Comment by austinkelleher** [comment]
Besides the comment I left above, everything else looks great. Thanks for making these changes. Let me know if you have any questions about the change I suggested above.

**Comment by newyork-anthonyng** [comment]
@austinkelleher I'm not seeing the comments that you left. Did you send it?

**Comment by austinkelleher** [comment]
@newyork-anthonyng Check now. I didn't submit the review.

**Comment by newyork-anthonyng** [comment]
@austinkelleher I dropped the commit that added the `routes` directory, and expanded the tutorial to clarify the routing. Let me know what you think.

**Comment by austinkelleher** [comment]
Thanks a lot for making these changes!

**Pull request merged**